client/scraper/wikipedia_metadata.py

```diff
@@ -1,16 +1,35 @@
-import collections
-import re
+from bs4 import BeautifulSoup
 from urlparse import urlparse
```

Comment:
urlparse is in python lib! (anything that is not in requirements should be in python lib)